rootstore/policy.md

```diff
@@ -364,12 +364,137 @@ set:
 
 *   RSA keys whose modulus size in bits is divisible by 8, and is at
     least 2048.
-*   Digest algorithms: SHA-1 (see below), SHA-256, SHA-384, or SHA-512.
-*   ECDSA keys using one of the following curve-hash pairs:
-    * P‐256 with SHA-256
-    * P‐384 with SHA-384
+*   ECDSA keys using one of the following curves:
+    * P-256
+    * P-384
 
-#### 5.1.1 SHA-1 ####
+The following sections detail encoding and signature algorithm requirements for
+each of these keys. The encoding requirements on signature algorithms apply to
+any contexts where the algorithm is encoded as an AlgorithmIdentifier,
+including:
+
+* The signatureAlgorithm field of a Certificate
+* The signature field of a TBSCertificate
+* The signatureAlgorithm field of a CertificateList
+* The signature field of a TBSCertList
+* The signatureAlgorithm field of a BasicOCSPResponse
+
+
+#### 5.1.1 RSA
+
+When RSA keys are encoded in a SubjectPublicKeyInfo structure, the algorithm
+field MUST consist of an rsaEncryption OID (1.2.840.113549.1.1.1) with a NULL
+parameter, as specified by [RFC 8017, Appendix A.1](https://tools.ietf.org/html/rfc8017#appendix-A.1)
+and [RFC 3279, Section 2.3.1](https://tools.ietf.org/html/rfc3279#section-2.3.1).
+The encoded AlgorithmIdentifier for an RSA key MUST match the
+following hex-encoded bytes:
+`300d06092a864886f70d0101010500`.
+
+CAs MUST NOT use the id-RSASSA-PSS OID (1.2.840.113549.1.1.10) within a
+SubjectPublicKeyInfo to represent a RSA key.
```

Comment:
Why not? I think it would be useful to have PSS-only public keys in certificates.